docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -25,7 +25,58 @@ For the supervised training we use the following algorithms components:
 
 - ``Loss function``: We use standart `Cross Entropy Loss <https://en.wikipedia.org/wiki/Cross_entropy>`_  to train a model.
 
-- Additionally, we use the **early stopping** to add adaptability to the training pipeline and prevent overfitting.
+- ``Training technique``
+    - ``Early stopping``: To add adaptability to the training pipeline and prevent overfitting. You can use early stopping like the below command.
+
+      .. code-block::
+
+        $ otx train {TEMPLATE} ... \
+            params \
+            --learning_parameters.enable_early_stopping=True \      # is early stopping used
+            --learning_parameters.early_stop_start=3 \              # the number of epochs (iters) in which early stopping proceeds
+            --learning_parameters.early_stop_patience=8 \           # (for epoch runner) stop if the model don't improve within the number of epochs of patience
+            --learning_parameters.early_stop_iteration_patience=8 \ # (for iter runner) stop if the model don't improve within the number of iterations of patience
+
+    - `Supervised Contrastive Learning (SupCon) <https://arxiv.org/abs/2004.11362>`_: To enhance the performance of the algorithm in case when we have a small number of data.
+      More specifically, we train a model with two heads: segmentation head with Cross Entropy Loss and contrastive head with `DetCon loss <https://arxiv.org/abs/2103.10957>`_.
+      The below table shows how much performance (mDice) SupCon improved compared with baseline performance on the subsets of Pascal VOC 2012 with two classes (person, car) and three classes (person, car, bicycle).
+      The number of the subset datasets is 8, 16, and 24.
+
+      `two classes (person, car)`
+
+      +--------------------+-------+--------+-------+--------+-------+--------+
+      | Model name         | #8    |        | #16   |        | #24   |        |
+      +====================+=======+========+=======+========+=======+========+
+      |                    | SL    | SupCon | SL    | SupCon | SL    | SupCon |
+      +--------------------+-------+--------+-------+--------+-------+--------+
+      | Lite-HRNet-s-mod2  | 60.36 | 60.86  | 63.75 | 66.84  | 66.43 | 69.81  |
+      +--------------------+-------+--------+-------+--------+-------+--------+
+      | Lite-HRNet-18-mod2 | 59.28 | 64.43  | 65.57 | 68.41  | 68.82 | 71.25  |
+      +--------------------+-------+--------+-------+--------+-------+--------+
+      | Lite-HRNet-x-mod3  | 59.98 | 63.36  | 63.60 | 68.63  | 68.33 | 71.21  |
+      +--------------------+-------+--------+-------+--------+-------+--------+
+
+      `three classes (person, car, bicycle)`
```

Comment:
I would suggest keeping only one table if it is okay, to not overwhelm with tables. For example, for the Semi-SL we also will choose only one label/unlabeled ratio. Is it possible to choose one table?

Reply:
Done